cmake/CPack.STGZ_Header.sh.in

```diff
@@ -1,35 +1,62 @@
 #!/usr/bin/env sh
+# vim: set tabstop=2:softtabstop=2:shiftwidth=2:noexpandtab
+
+# Reset
+Color_Off='\033[0m'				# Text Reset
+URed='\033[4;31m'					# Red
```

Comment:
Are these valid for non-bash shells?  This script's shebang line is attempting to run `sh` not `bash`.  I have been searching for non-bash color output codes and literally everything is focusing on bash terminals.  This could be because most people think of bash as the only terminal on Linux, so the answers are confusing online, or it's because the color codes really are specific to bash.  Anyway, please verify.  If we aren't sure, then I'm inclined to turn the color back off to avoid breaking other shells.

Reply:
It works for `sh` (dash) on ubuntu 16.04 at least as I tested the installer, but see a MCVE below.

```
URed='\033[4;31m'
Color_Off='\033[0m'
sh -c "echo 'coucou ${URed}COUCOU${Color_Off}'"
```

![image](https://user-images.githubusercontent.com/5479063/58476295-a6f5bc80-8150-11e9-84bc-bd20b6a45030.png)

**I don't care if you want me to remove them though, just let me know.**

Reply:
After doing some research, it looks like we are on the up and up.  According to Debian policy, any script that includes a shebang line with `sh` must simply be POSIX-compliant, and the color codes are POSIX-fine.  So that's good.  If anything were to change, I'd consider making the red code _not_ underlined because it looks like it is a hyperlink.  I'd go with bold red (1;31m).

Reply:
nside dash directly:

![image](https://user-images.githubusercontent.com/5479063/58477971-31d8b600-8155-11e9-82df-2ce4b9f27566.png)

Reply:
Done, replaced with 
```
BRed='\033[1;31m'				# Bold Red
```

Will push after testing on mac.